spec/lib/geocode_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe Geocode do
+  it 'normalises the postcode' do
+    expect(Geocoder).to receive(:coordinates).with('RG11GG, United Kingdom')
+
+    Geocode.call('rg1 1gg')
+  end
+
+  it 'returns the lat/long pair' do
+    VCR.use_cassette(:rg11gg) do
+      latitude, longitude = Geocode.call('RG11GG')
+
+      expect(latitude).to  be
+      expect(longitude).to be
```

Comment:
I'd side with @tomas-stefano. Currently, your case description read `returns the lat/long pair`, while your test checks only if the method returns two values.

I'd personally provide doubles for lat / long response from Geocode and test if method return values are the same.